oidc-provider/simple_op/src/provider/authn/__init__.py

```diff
@@ -0,0 +1,30 @@
+import importlib
```

Comment:
All this code is a direct copy from the examples in pyoidc. I think that ideally we should leave it as is to prevent anything like a fork. In the best possible world I'd not check the code in here, but instead have a script that fetches pyoidc and extracts this example (the docker version did this). @jeromekelleher would this make more sense to you?